app/main/RTMediaUploadTerms.php

```diff
@@ -0,0 +1,127 @@
+<?php
+/**
+ * This file applies the admin settings and handle feature of Upload Terms at front-end side.
+ *
+ * @package rtMedia
+ * @author  Malav Vasita <malav.vasita@rtcamp.com>
+ */
+include_once( ABSPATH . 'wp-admin/includes/plugin.php' );
```

Comment:
`include_once` is a statement, not function, if code below is dependent on this file then use `require_once`